test/unit/material-icon/index.test.tsx

```diff
@@ -26,15 +36,26 @@ test('has icon type as child', () => {
   const wrapper = mount(<MaterialIcon icon='menu' />);
   assert.equal(wrapper.find('.material-icons').text(), 'menu');
 });
+
 const rippledIconComponent = (
-  <RippleMaterialIcon hasRipple unbounded icon='menu' />
+  <RippleMaterialIcon hasRipple unbounded icon='menu' tag='i' />
+);
+
+const rippledTaggedIconComponent = (
+  <RippleMaterialIcon hasRipple unbounded icon='menu' tag='a' />
 );
 
 test('if hasRipple true, then it should contain RippleMaterialIcon', () => {
   const wrapper = mount(<MaterialIcon icon='menu' hasRipple />);
   assert.isTrue(wrapper.contains(rippledIconComponent));
 });
 
+test('if hasRipple true prop.tag, then it should contain RippleMaterialIcon with custom tag', () => {
```

Comment:
can you add a test that adds an href? I think the typescript compiler might error.

Reply:
Oh yes, it does throw an error. I've been a bit busy with other work the past week. I'm probably going have to delay this PR for a bit. I'll try to get it set for the next release. Do you have a date in mind for that?

Reply:
Sounds good! Milestone 12 is going to happen April 16th.